burr/integrations/persisters/b_mongodb.py

```diff
@@ -130,6 +130,27 @@ def save(
     def __del__(self):
         self.client.close()
 
+    def __getstate__(self) -> dict:
+        state = self.__dict__.copy()
+        state["connection_params"] = {
```

Comment:
Can we shift this up to a higher level in the interface? E.G. let the persistors tell us how to serialize/deserialize their connection params? Not needed now but would make it easier to implement.

Reply:
sure. done.